### PR TITLE
Add govuk-nav-prototype application

### PR DESCRIPTION
**Pull request by Davidslv**
We need to access the draft-content-store in our prototype to display
draft taxonomies without publishing the taxonomy live (doing so could break user journeys and we don't want that).

Currently we can't access the draft-content-store from the "outside" (e.g: heroku) so we decided to add the prototype alongside other apps.

Trello: https://trello.com/c/9leIHFaX/230-deploy-prototype-to-govuk-infrastructure
Paired with @alecgibson

### CODE REVIEW

File: modules/govuk/manifests/apps/govuk_nav_prototype.pp [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# == Class: govuk::apps::govuk_nav_prototype
+#
+# This is a the navigation prototype application
+# Read more: https://github.com/alphagov/govuk-nav-prototype
+#
+# === Parameters
+#
+# [*port*]
+#   The port that govuk-nav-prototype is served on.
+#   Default: 3071
```
Comment (surminus) [inline modules/govuk/manifests/apps/govuk_nav_prototype.pp]: this needs updating

File: modules/govuk/manifests/apps/govuk_nav_prototype.pp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#
+# [*enable_procfile_worker*]
```
Comment (surminus) [inline modules/govuk/manifests/apps/govuk_nav_prototype.pp]: this parameter doesn't exist

**Comment by alecgibson** [comment]
@surminus I've addressed your comments - could you please re-review?

File: modules/govuk/manifests/apps/govuk_nav_prototype.pp [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+#
+# === Parameters
+#
+# [*port*]
+#   The port that govuk-nav-prototype is served on.
+#   Default: 3211
+#
+class govuk::apps::govuk_nav_prototype (
+  $port = 3211,
+  $secret_key_base = undef,
```
Comment (surminus) [inline modules/govuk/manifests/apps/govuk_nav_prototype.pp]: We should add this to puppetdocs

Comment (alecgibson) [inline modules/govuk/manifests/apps/govuk_nav_prototype.pp]: @surminus what is puppetdocs?

**Comment by surminus** [comment]
If you want to be able to deploy the application through Jenkins you'll need to add your application to yet another list [here](https://github.com/alphagov/govuk-puppet/blob/master/hieradata/common.yaml#L9). 

This will also create stuff in CI which will allow you to run tests in Jenkins and deploy automatically to Integration on master assuming you have a Jenkinsfile in the repo.

**Comment by alecgibson** [comment]
@surminus thanks - I've added that  too.

File: modules/govuk/manifests/apps/govuk_nav_prototype.pp (hunk 3 above)
Comment (surminus) [inline modules/govuk/manifests/apps/govuk_nav_prototype.pp]: Sorry, I meant just the docs above (puppetdocs is the format).

eg
```
[*secret_key_base*]
...
```

**Pull request merged**